wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/ast2wdlom/AstNodeToExpressionElement.scala

```diff
@@ -14,26 +17,196 @@ import wom.values._
 import scala.util.Try
 
 object AstNodeToExpressionElement {
+
   def convert(astNode: AstNode): ErrorOr[ExpressionElement] = astNode match {
 
     case t: Terminal if asPrimitive.isDefinedAt((t.getTerminalStr, t.getSourceString)) => asPrimitive((t.getTerminalStr, t.getSourceString)).map(PrimitiveLiteralExpressionElement)
-    case t: Terminal => s"No rule available to create ExpressionElement from terminal: ${t.getTerminalStr} ${t.getSourceString}".invalidNel
+    case t: Terminal if t.getTerminalStr == "identifier" => IdentifierLookup(t.getSourceString).validNel
+
+    case a: Ast if lhsRhsOperators.contains(a.getName) => useValidatedLhsAndRhs(a, lhsRhsOperators(a.getName))
+    case a: Ast if unaryOperators.contains(a.getName) => a.getAttributeAs[ExpressionElement]("expression").map(unaryOperators(a.getName)).toValidated
+    case a: Ast if a.getName == "TupleLiteral" => (a.getAttributeAsVector[ExpressionElement]("values") flatMap {
+      case pair if pair.length == 2 => PairLiteral(pair.head, pair(1)).validNelCheck
```

Comment:
Why not pair.tail instead of pair(1)?

Reply:
`pair.tail` will return a `Vector[ExpressionElement]` representing everything except the `head`. But I only want the second element of the original Vector.

I could do `pair.tail.head` but it's easier to just go straight to the item at index 1

Reply:
Makes sense! Thanks for the explanation.